src/App.js

```diff
@@ -69,16 +69,25 @@ class App extends React.Component {
     });
   };
   clearDone = event => {
-    // console.log("cleardone");
-    // event.preventDefault();
-    // this.setState({
-    //   lists: this.state.lists.map(item => item + "hello")
-    // });
+    // console.log("clearDone");
+    event.preventDefault();
+    const doneList = {
+      task: this.state.task,
+      id: Date.now(),
+      completed: false
+    };
+    doneList.filter(item => {
```

Comment:
why are we creating a doneList here? your making an object and trying to run an array method on it.

this would be a good spot for a .filter or .map function.
we can just run this on our state task list.
```
const notDoneList = this.state.lists.filter((item) =>{
     return !item.completed
})

this.setState({
    lists: notDoneList
})
```